js/common.js

```diff
@@ -155,9 +155,9 @@ function displayOtherSelectOptions(select_object, other_option_name) {
  * @param    reference_id    DOM element
  * @param    container_id    DOM element
 **/
-function checkAsCheckboxes(reference_id, container_id) {
+function checkAsCheckboxes(reference, container_id) {
     $('#' + container_id + ' input[type="checkbox"]:enabled')
```

Comment:
To ensure there is no BC break, maybe passing reference_id should be supported.
 ```suggestion
    reference = typeof(reference) === 'string' ? document.getElementById(reference) : reference;
    $('#' + container_id + ' input[type="checkbox"]:enabled')
```